test/module/irohad/torii/torii_queries_test.cpp

```diff
@@ -146,10 +143,13 @@ TEST_F(ToriiQueriesTest, FindAccountWhenNoGrantPermissions) {
   account.account_id = "b@domain";
   auto creator = "a@domain";
 
+  EXPECT_CALL(*storage, getWsvQuery()).WillRepeatedly(Return(wsv_query));
```

Comment:
Same as before, since there are no tests that return `nullptr` for `WsvQuery` or `BlockQuery`, this can be moved to `SetUp`.